bazel/repository_locations.bzl

```diff
@@ -357,6 +357,19 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2021-01-26",
         cpe = "N/A",
     ),
+    com_github_aws_aws_xray_daemon = dict(
```

Comment:
This is not necessary. these repository locations are for code dependencies that envoy needs to build and run.

Reply:
I thought this section refers to possible external dependancies that are needed for envoy to work. I will remove this hunk entirely then.

Reply:
Fixed in the latest commit